apps/graphql/src/dataloaders/itinerariesHelpers.js

```diff
@@ -151,17 +151,17 @@ const apiRouteItemToDeparture = (routeItem: ?ApiRouteItem) => ({
 
 const sanitizeRouteStop = ({
   cityName,
-  cityId,
+  code,
   time,
 }: {|
   +cityName: ?string,
-  +cityId: ?string,
+  +code: ?string,
   +time: {| +utc: ?string, +local: ?string |},
 |}): RouteStop => ({
   cityName,
-  cityId,
+  cityId: null,
```

Comment:
Why is `cityId` not removed from the `RouteStop`?

Reply:
Because tequila does not return the cityId, it returns the iata code of the stop.

Reply:
We have cityId from the mocked data from MMB, though we are currently not using it, so I guess I can remove it, and add it later if we need it

Reply:
Sorry, scratch that, I use it to get `destinationImageUrl` in `BookingInterface`. So cityId has to stay in the resolvers.